examples/crowdfund.vy

```diff
@@ -36,6 +44,15 @@ def finalize():
 
     selfdestruct(self.beneficiary)
 
+#vyper.exceptions.InvalidTypeException: Invalid base type: Funder
+#@private
+#def getFunderAddress(f: Funder) -> address:
+#  return f.sender.x
```

Comment:
Structs in function args was a different VIP because you have to get into the ABI implementation.

Reply:
Thanks for the note! I was documenting the error it currently throws.